docs/sphinx/sections/api/apidocs/libraries/dagster-dbt.rst

```diff
@@ -121,3 +121,7 @@ Utils
 .. currentmodule:: dagster_dbt.utils
 
 .. autofunction:: generate_materializations
+
+.. autofunction:: default_group_fn
```

Comment:
Do we have a better name for this? Don't like this `_fn` prefix. Maybe `_from_dbt_resource_props`?

- `default_group_from_dbt_resource_props`
- `folder_group_from_dbt_resource_props`
- `directory_group_from_dbt_resource_props`
- `subdirectory_group_from_dbt_resource_props`